src/components/TableDialog.vue

```diff
@@ -37,6 +91,7 @@ import Vue from 'vue';
 
 import api from '@/api';
 import { FileType } from '@/types';
+import { validFileType, fileName as getFileName } from '@/utils/files';
 import FileUploadForm from '@/components/FileUploadForm.vue';
```

Comment:
Can you remove this import now?

Reply:
What do you mean? They're being used in the file

Reply:
I should have been more specific. I mean the `FileUploadForm` since you removed that component and implemented the upload form here.

Reply:
Oh my bad, I thought you meant the line above, yes I'll remove that.